core/capabilities/compute/cache_test.go

```diff
@@ -62,7 +62,7 @@ func TestCache_EvictAfterSize(t *testing.T) {
 	tick := 1 * time.Second
 	timeout := 1 * time.Second
 
-	cache := newModuleCache(clock, tick, timeout, 1)
+	cache := newModuleCache(clock, tick, timeout, 0)
```

Comment:
modules are only evicted from the cache > the last parameter, since only one value was added then the module was never evicted.